snowflake/table_snowflake_resource_monitor.go

```diff
@@ -0,0 +1,92 @@
+package snowflake
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+//// TABLE DEFINITION
+
+func tableSnowflakeResourceMonitor(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "snowflake_resource_monitor",
+		Description: "Lists all the resource monitors in your account for which you have access privileges.",
+		List: &plugin.ListConfig{
+			Hydrate: listSnowflakeResourceMonitors,
+		},
+		Columns: snowflakeColumns([]*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "Name for warehouse."},
+			{Name: "credit_quota", Type: proto.ColumnType_DOUBLE, Description: "Monthly credit quota for the resource monitor."},
+			{Name: "used_credits", Type: proto.ColumnType_DOUBLE, Description: "Number of credits used in the current monthly billing cycle by all the warehouses associated with the resource monitor."},
+			{Name: "remaining_credits", Type: proto.ColumnType_DOUBLE, Description: "Number of credits still available to use in the current monthly billing cycle."},
+			{Name: "level", Type: proto.ColumnType_STRING, Description: "Level"},
+			{Name: "frequency", Type: proto.ColumnType_STRING, Description: "Daily, Weekly, etc"},
+			{Name: "start_time", Type: proto.ColumnType_TIMESTAMP, Description: "Date and time when the monitor was started."},
+			{Name: "end_time", Type: proto.ColumnType_TIMESTAMP, Description: "Date and time when the monitor was stopped."},
+			{Name: "notify_at", Type: proto.ColumnType_STRING, Description: "Levels to which to alert."},
+			{Name: "suspend_at", Type: proto.ColumnType_STRING, Description: "Levels to which to suspend warehouse"},
+			{Name: "suspend_immediately_at", Type: proto.ColumnType_STRING, Description: "Levels to which to suspend warehouse"},
```

Comment:
Does this column's value ever have more than 1 level at which the warehouse will be suspended? If so, is it a comma delimited list (or a list with another delimiter)? Going off of the description, it seems like we could expect more than 1 level.

Do you have a link to where you pulled this description from? That may be helpful for me to compare the current descriptions vs. what the docs/other sources mention.

Reply:
This column can have 1 or more values, having problems compiling the code at the moment, the function keeps show the show warehouse schema

Reply:
@paulsewardasc Are you still having troubles compiling the plugin? Please let me know here or over Slack if there's anything I can help with.

Reply:
Fixed it, the plugin.go file was wrong!

Reply:
The credit_quota is defined in snowflake as a variant data type.

Reply:
The remaining_credits is defined as a float and suspend is a NUMBER(38,0) is that helps?

Reply:
READER_ACCOUNT_NAME	VARCHAR(16777216)
NAME	VARCHAR(16777216)
CREATED	TIMESTAMP_LTZ(6)
CREDIT_QUOTA	VARIANT
USED_CREDITS	VARIANT
REMAINING_CREDITS	FLOAT
OWNER	VARCHAR(16777216)
WAREHOUSES	VARCHAR(16777216)
NOTIFY	NUMBER(38,0)
SUSPEND	NUMBER(38,0)
SUSPEND_IMMEDIATE	NUMBER(38,0)
LEVEL	VARCHAR(9)
READER_ACCOUNT_DELETED_ON	TIMESTAMP_LTZ(6)

Reply:
For the `credit_quota` column, I think OK to leave as a double. In the example query results you added, they appear to be integers instead of doubles, but if there's a possibility there's precision as per https://docs.snowflake.com/en/sql-reference/data-types-numeric.html#number, then double is correct. 

For `notfiy_at`, `suspend_at`, and `suspend_at_immediately`, do you have example query results that include single value and multi-value rows? I'd be curious to see how the data looks since the column types are currently marked as STRING, but if there can be multiple values, should the column types be JSON instead (which handles arrays)?